backend/src/main/scala/cromwell/backend/RuntimeEnvironment.scala

```diff
@@ -27,7 +27,8 @@ object RuntimeEnvironmentBuilder {
       callRoot.resolve(s"tmp.$hash").pathAsString
     }
 
-    val cores: Int Refined Positive = CpuValidation.instanceMin.validate(runtimeAttributes).getOrElse(minimums.cores)
+    // This class is going away in https://github.com/broadinstitute/cromwell/pull/7369
```

Comment:
...and this statement needed updating to keep these PRs compiling while in separate branches (incredibly, the only overlap)